translations/en-GB.json

```
@@ -1409,13 +1409,13 @@
             },
             "NOT_MATCH_REGEX": {
               "NAME": "not-match-regex",
-              "DESCRIPTION": "For advanced matching you can provide one regex pattern instead of not-include-text. (e.g. /^http:/)"
+              "DESCRIPTION": "For advanced matching, you can provide one regex pattern instead of not-include-text. (e.g. /^http:/)"
```

```suggestion
              "DESCRIPTION": "For advanced matching you can provide one regex pattern instead of not-include-text. (e.g. /^http:/)"
```
comma exceeds the 100 char limit 😅